diagon-alley/src/Pages/Authentication/ForgotPassword.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { ForgotPasswordForm, HomeWallpaper, Themes, TopNav } from '../../Components';
+
+const ForgotPassword = () => {
+  return (
+    <>
+      <HomeWallpaper /> 
+      <TopNav showHamburger={false}/>
+      <Themes />
```

Comment:
Just a Suggestion since these Three Components are repeating you can put it in auth page and just change the remaining one's

Reply:
Okay will change, I was also thinking of doing it, thank you